src/tests/user.test.js

```diff
@@ -26,6 +26,25 @@ describe('/POST Signup route', () => {
       });
   });
 
+  it('should return an error if email is not in the right format', done => {
```

Comment:
You can remove this test. It has been covered from line 10 to 27 of this file.

Reply:
thanks